bar.go

```diff
@@ -43,6 +43,14 @@ func (b *bar) WindowAdded(win Window) {
 	icon := barCreateIcon(b, true, data, win)
 	if icon != nil {
 		icon.onTapped = func() {
+			if !win.Iconic() && win.TopWindow() {
```

Comment:
This is getting complex, it should probably be a separate function (named, not inline).
Also why does it only iconify top level windows?

Reply:
Oh yes, I thought this was the iconify button on the frame, my bad